### PR TITLE
Add Option for Change Button Color as Per Theme

### PR DESCRIPTION
**Pull request by harpreet9629**
Screenshot_1
Screenshot_2
Screenshot_3
Screenshot_4

It make looks nice :)

### CODE REVIEW

**Comment by teambrixmusic** [comment]
YES! THANK YOU!!!

**Comment by teambrixmusic** [comment]
@harpreet9629 This is awesome and love the change with the select BAR (under tabs) and button possibly matching everything. Love the added --button also. However, I think you may have accidentally modified cake-theme.ts "Link" color to dark purple which is too similar to the regular text. The changes to Cake effectively remove all blue, which was custom to the theme (and not just because the button option wasn't available.) Love that future themes would be able to implement this option <3

If this merges I'll probably end up pulling again to readjust the cake theme per the original. Any way you could just revert the link 009aff , hover 008cad and new button 009aff back to blue since it isn't merged yet?

Love the gray theme too!

**Comment by harpreet9629** [comment]
Done i revert theme colors back!

**Comment by teambrixmusic** [comment]
> Done i revert theme colors back!
@harpreet9629 
I love what you've done, by the way! That opens things up for future theming options which is awesome. I too am frustrated by the limited number of class options. I thought it would also be simple someday to implement other google fonts since the themes primarily use Roboto. Also might be cool to have avatars someday square | rounded | round but I haven't looked at everything.

Do you have a screenshot of gray?

**Comment by harpreet9629** [comment]
Screenshot_5
It's Green not gray, i will change theme name.

File: src/styles.scss [hunk 1]
```diff
@@ -139,15 +139,15 @@ $border-grey: #E5E5E5;
 $staking-grey: #F0F0F0;
 $light-grey-for-divider: #F8F8F8;
 $light-grey-for-divider-2: #FAFAFA;
-$buy-button-blue: #0058F7;
```
Comment (lazynina) [inline src/styles.scss:142]: let's use this hex value as the button color for light, dark, and icy dark themes.

File: src/styles.scss [hunk 2]
```diff
@@ -1999,10 +1999,20 @@ input {
...
+  background-color: $buy-button-blue;
+  border-color: $buy-button-blue;
   color: #fff;
 }
+.btn-primary.disabled, .btn-primary:disabled {
+	color: #fff;
+	background-color: $buy-button-blue;
+	border-color: $buy-button-blue;
+}
+.btn-primary:hover {
```
Comment (lazynina) [inline src/styles.scss:2011]: for some reason, when you hover over a disabled button, it changes color, which it shouldn't do when disabled.

**Review by lazynina — CHANGES_REQUESTED** [review]

File: src/app/theme/gray-theme.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { Theme } from "./symbols";
```
[the remark this answers is not in the document]
Comment (harpreet9629) [reply src/app/theme/gray-theme.ts:1]: Ok i will do remove

File: src/styles.scss (hunk 2 above)
Comment (harpreet9629) [reply src/styles.scss:2011]: Ok got it cause hover is after disabled class

File: src/styles.scss (hunk 1 above)
Comment (harpreet9629) [reply src/styles.scss:142]: Ok sure

**Comment by harpreet9629** [comment]
@lazynina its fixed 👍

**Review by lazynina — APPROVED** [review]

**Pull request merged**